src/plugins/vis_type_timeseries/public/application/components/lib/index_pattern_select/switch_mode_popover.tsx

```diff
@@ -63,17 +72,34 @@ export const SwitchModePopover = ({ onModeChange, useKibanaIndices }: PopoverPro
           />
         </EuiText>
         <EuiSpacer />
-        <EuiSwitch
-          checked={useKibanaIndices}
-          label={i18n.translate(
-            'visTypeTimeseries.indexPatternSelect.switchModePopover.useKibanaIndices',
-            {
-              defaultMessage: 'Use only Kibana index patterns',
-            }
-          )}
-          onChange={switchMode}
-          data-test-subj="switchIndexPatternSelectionMode"
-        />
+        <EuiToolTip
+          content={
+            isSwitchDisabled && (
+              <FormattedMessage
+                id="visTypeTimeseries.indexPatternSelect.switchModePopover.tooltip"
+                defaultMessage="Using Elasticsearch indices is not recommended.
+      To use index patterns, go to {advancedSettings}, then enable {allowStringIndices}."
+                values={{
+                  advancedSettings: <strong>Advanced Settings</strong>,
+                  allowStringIndices: <strong>Allow string indices in TSVB</strong>,
```

Comment:
shouldn't these strings be i18n as well?

Reply:
@dej611 you are right, @DziyanaDzeraviankina please also do the same for `advancedSettings`